test/has_scope_test.rb

```diff
@@ -50,7 +50,7 @@ def show_all_colors?
       false
     end
 
-    if ActionPack::VERSION::MAJOR == 5
+    if ActionPack::VERSION::MAJOR >= 5
```

Comment:
💅 Can we use a method to encapsulate this logic? We can use in [this line](https://github.com/plataformatec/has_scope/pull/109/files#diff-444bb395eb6152af5506a657fbdd0d30R337) too, It will be useful in the next changes in this file.

Reply:
Since we only use this logic here on this file and just two times (this remembers me about the [Rule of Three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)) 😄) I'm not that sure if the benefits outweigh the costs of creating a new method (deciding where to put it, what to name it, etc...).

WDYT? 🤔

Reply:
I commented on this because the next one may forget to change one of the clauses. But no problems, you have a good point. You decide bro 😉